### PR TITLE
Add addon and evolution callout

### LINKED ISSUE #4224: [Devhub Landing] Implement Manage Your Add-on and Evolution block (signed-out)
Mocks looks like this:

d94a454a-aca8-11e6-852f-92284eda5db0_png__png_image__1280 x 2850_pixels_

710ffb0e-aca9-11e6-8eeb-6a75df8e0da2_png__png_image__320 x 2860_pixels_

### PR DESCRIPTION
**Pull request by muffinresearch**
Fixes #4224 

I still need to add the ladder image, but I think this is mostly ready otherwise. 

Full spec tightening + cleanups can be a follow-up too.

Looks like this: 

developer_hub____add-ons_for_firefox

developer_hub____add-ons_for_firefox

developer_hub____add-ons_for_firefox

developer_hub____add-ons_for_firefox

### CODE REVIEW

File: static/css/devhub/new-landing/base.less [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
 // Grid
 @grid-max: 940px;
-@extra-small: ~"only screen and (max-width: 599px)";
-@small: ~"only screen and (min-width: 599px)";
+@extra-small: ~"only screen and (max-width: 399px)";
+@small: ~"only screen and (min-width: 400px)";
 @medium: ~"only screen and (min-width: 700px)";
 @large: ~"only screen and (min-width: 1024px)";
-@retina: ~"only screen and (min-device-pixel-ratio: 2)";
+@retina: ~"only screen and (min-resolution: 1.5dppx)";
```
Comment (muffinresearch) [inline static/css/devhub/new-landing/base.less]: This is the proper way to do resolution queries since min-device-pixel-ratio is old hat.

Comment (EnTeQuAk) [inline static/css/devhub/new-landing/base.less]: Heh, I keep learning :)

File: static/css/devhub/new-landing/common.less [hunk 2]
```diff
@@ -71,25 +71,85 @@
...
+  right: 0;
+}
 
 .DevHub-content-copy {
   position: relative;
   z-index: 4;
 }
 
+.DevHub-callout-box {
+  border: 8px solid #FDC900;
```
Comment (EnTeQuAk) [inline static/css/devhub/new-landing/common.less]: I wonder if this color code should be extracted into a variable.

File: static/css/devhub/new-landing/common.less [hunk 3]
```diff
@@ -71,25 +71,85 @@
...
+  background: #fff;
+  padding: 0 20px 20px;
+  // Override default padding on container
+  // with negative margin.
+  margin: 0 -20px -20px;
+
+  @media @medium {
+    box-shadow: 8px 8px 0 0 rgba(0,0,0,0.15);
```
Comment (EnTeQuAk) [inline static/css/devhub/new-landing/common.less]: whitespaces after comma in rgba section?

File: static/css/devhub/new-landing/common.less [hunk 4]
```diff
@@ -71,25 +71,85 @@
...
+}
+
+.DevHub-content-header--manage {
+  max-width: 100%;
+  text-align: center;
+}
+
+.DevHub-content-container--manage {
+  flex-direction: column;
+  background-image: url(../../../img/developers/new-landing/manage-addon.png);
```
Comment (EnTeQuAk) [inline static/css/devhub/new-landing/common.less]: Interesting that the highlighting doesn't work but I don't see any syntax problem.

Comment (muffinresearch) [inline static/css/devhub/new-landing/common.less]: Yeah a lot of CSS syntax grammar seems to trail, I have similar issues with the newer stuff.

File: static/css/devhub/new-landing/common.less (hunk 2 above)
Comment (muffinresearch) [inline static/css/devhub/new-landing/common.less]: Yeah, we're going to need this more. I'll sort that.

**Pull request merged**